modules/catalog/src/main/java/org/apache/ignite/internal/catalog/events/AlterZoneEventParameters.java

```diff
@@ -23,25 +23,25 @@
  * Alter zone event parameters contains a distribution zone descriptor for newly created distribution zone.
  */
 public class AlterZoneEventParameters extends CatalogEventParameters {
-
-    private final CatalogZoneDescriptor zoneDescriptor;
+    private final CatalogZoneDescriptor descriptor;
```

Comment:
Same here. Why did you rename all these fields?

Reply:
I won't be leaving the same comment over and over again. I see these renames multiple times later.

Reply:
But not for me.